src/table/cells/handle-cell-focus.js

```diff
@@ -80,3 +80,53 @@ export const handleNavigateTop = ({ tableSection, focusedCellCoord, rootElement
     }
   }
 };
+
+const getCellSelectionStatusNote = (rows) => {
+  if (!Array.isArray(rows) || !rows.length) return '';
```

Comment:
This will never happen since the call is inside `if (selstate.rows.length)` already

Reply:
✅ Fixed